js/expose.js

```diff
@@ -0,0 +1,43 @@
+var chromedpExposeFunc = chromedpExposeFunc || {
+  bindings: {},
+  deliverError: function (name, seq, message) {
+    const error = new Error(message);
+    chromedpExposeFunc.bindings[name].callbacks.get(seq).reject(error);
+    chromedpExposeFunc.bindings[name].callbacks.delete(seq);
+  },
+  deliverResult: function (name, seq, result) {
+    chromedpExposeFunc.bindings[name].callbacks.get(seq).resolve(result);
```

Comment:
1. I'm not sure that `CallFunctionOn`  param `ExecutionContextID` can or not  avoid this problem. 
2. I agree to check the existence of seq to avoid js exceptions.
3. Is UUID too complex? Puppeteer doesn't handle this situation.  The callback is also specified to  `event.executionContextId`.
  ```
Puppeteer #onBindingCalled

this.#client
      .send('Runtime.evaluate', {
        expression,
        contextId: event.executionContextId,
      })
      .catch(debugError);
```

Reply:
You're correct. `ExecutionContextID` can prevent this problem. My bad. So we don't need UUID (BTW, when I wrote UUID, I just meant a unique id to some extend. For example, `Math.random()` should be enough).